Cargo.toml

```diff
@@ -42,14 +42,14 @@ rust-version = "1.77.1"
 anyhow = "1.0.72"
 apache-avro = "0.17"
 array-init = "2"
-arrow-arith = { version = "54.1.0" }
-arrow-array = { version = "54.1.0" }
-arrow-buffer = { version = "54.1.0" }
-arrow-cast = { version = "54.1.0" }
-arrow-ord = { version = "54.1.0" }
-arrow-schema = { version = "54.1.0" }
-arrow-select = { version = "54.1.0" }
-arrow-string = { version = "54.1.0" }
+arrow-arith = { version = "54.2.0" }
+arrow-array = { version = "54.2.0" }
+arrow-buffer = { version = "54.2.0" }
+arrow-cast = { version = "54.2.0" }
+arrow-ord = { version = "54.2.0" }
+arrow-schema = { version = "54.2.0" }
+arrow-select = { version = "54.2.0" }
+arrow-string = { version = "54.2.0" }
```

Comment:
MSRV failed. So we need to bump arrow crates
<img width="1492" alt="image" src="https://github.com/user-attachments/assets/4067c930-96ee-4b67-879d-fd1849ec2113" />